### DISCUSSION TITLE
Is this HTMX version for VUE?

### DISCUSSION THREAD

**Discussion by dante01yoon** [vuejs/petite-vue#221] [General]
Greeting!

I wonder difference between HTMX

**Comment by sqllyw** [comment]
almost like that:) I took a look at htmx, and came back to petite-vue

**Comment by sesm** [comment]
The main difference is that HTMX is primarily an "HTML-over-the-wire" framework like Rails Turbolinks, PJAX, etc, while petite-vue is not doing any of that an re-implements early versions of Angular 1 while intentionally keeping a small scope.
